### PR TITLE
added section01

### PR DESCRIPTION
**Pull request by OlegAdamovich**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        <nav class="nav">
+            <div class="nav__item">Home</div>
+            <div class="nav__item">About</div>
+            <div class="nav__item">Services</div>
+            <div class="nav__item">Portfolio</div>
+            <div class="nav__item">Contact</div>
+        </nav>
+    </header>
+
+    <section class="section1">
```
Comment (mhryshkin) [inline index.html:32]: @OlegAdamovich When you add new sections, is it gonna be `section 2`, `section 3`, `section 4` etc.? If I ask you then to change the order of section, what happens?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        <div class="container">
```
Comment (mhryshkin) [inline index.html:33]: @OlegAdamovich Do you need an extra `div` element here? Look here - https://ru.bem.info/methodology/quick-start/#%D0%BC%D0%B8%D0%BA%D1%81

any conclusions?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            <div class="section1__image">
+                <img src="images/phone.png" alt="image">
+            </div>
+            <div class="section1__content">
+                <div class="section1__text">Lorem ipsum dolor sit amet</div>
+                <div class="section1__text_bold">Donec nec justo eget</div>
+                <div class="section1__text_regular-gray">
```
Comment (mhryshkin) [inline index.html:40]: Is this the only place in the whole document you will have `text_regular-gray`? How are you gonna reuse it? Using `section1__text_regular-gray`?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                    Morbi in sem quis dui placerat ornare. Pellentesque odio
+                    euismod in, pharetra a, ultricies in, diam.
+                </div>
+                <button class="section1__button">More Info</button>
```
Comment (mhryshkin) [inline index.html:44]: Same as above - `section1__button` I hardly believe this the only button in the whole document, how will you reuse the button styles?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    <title>Document</title>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;700&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet" href="../style.css">
+</head>
+<body>
+<div class="wrapper">
```
Comment (mhryshkin) [inline index.html:17]: Is it necessary to have here an extra div?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;700&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet" href="../style.css">
```
Comment (mhryshkin) [inline index.html:14]: Do you connect styles twice?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+</head>
+<body>
+<div class="wrapper">
+
+    <header class="header">
+        <div class="header__logo">
+            <img src="images/logo.png" alt="logo">
+        </div>
+        <nav class="nav">
+            <div class="nav__item">Home</div>
```
Comment (mhryshkin) [inline index.html:24]: Are you it has to be a div? I think you will need some anchors later

File: index.html [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+            <div class="nav__item">Services</div>
+            <div class="nav__item">Portfolio</div>
+            <div class="nav__item">Contact</div>
+        </nav>
+    </header>
+
+    <section class="section1">
+        <div class="container">
+            <div class="section1__image">
+                <img src="images/phone.png" alt="image">
```
Comment (mhryshkin) [inline index.html:35]: `alt=image` looks really 'helpful', especially if a user doesn't see an image

**Review by mhryshkin — CHANGES_REQUESTED** [review]

File: index.html (hunk 8 above)
Comment (OlegAdamovich) [reply index.html:35]: Rename attribute alt that helps to understanding the content.

File: index.html (hunk 7 above)
Comment (OlegAdamovich) [reply index.html:24]: Changed `div` to tag `a`.

File: index.html (hunk 6 above)
Comment (OlegAdamovich) [reply index.html:14]: Extra link removed.

File: index.html (hunk 5 above)
Comment (OlegAdamovich) [reply index.html:17]: I don't understand why this div is extra. It sets the width and position of the document.

File: index.html (hunk 4 above)
Comment (OlegAdamovich) [reply index.html:44]: Changed it in independent page component that can be reused.

File: index.html (hunk 5 above)
Comment (mhryshkin) [reply index.html:17]: @OlegAdamovich Doesn't the body tag work for you?

File: index.html (hunk 2 above)
Comment (OlegAdamovich) [reply index.html:33]: Got it. Extra div removed.

File: index.html (hunk 1 above)
Comment (OlegAdamovich) [reply index.html:32]: Renamed.

File: index.html (hunk 8 above)
Comment (mhryshkin) [reply index.html]: @OlegAdamovich `alt` attribute is shown to a user, don't be shy to write in a more readable way like 'Phone image' and 'Logo icon'

File: index.html [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+    </header>
+
+    <section class="description">
+        <div class="description__image">
+            <img src="images/phone.png" alt="phone-image">
+        </div>
+        <div class="description__content">
+            <div class="description__content_text">Lorem ipsum dolor sit amet</div>
+            <div class="description__content_text-bold">Donec nec justo eget</div>
+            <div class="text-gray description_text">
```
Comment (mhryshkin) [inline index.html:38]: What is `description_text`? It looks like a modifier

File: index.html (hunk 5 above)
Comment (OlegAdamovich) [reply index.html:16]: Wrapper removed. Add styles to `body`.

File: index.html (hunk 9 above)
Comment (OlegAdamovich) [reply index.html:38]: Converted in element.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+    </header>
+
+    <section class="description">
+        <div class="description__image">
+            <img src="images/phone.png" alt="phone-image">
+        </div>
+        <div class="description__content">
+            <div class="description__content_text">Lorem ipsum dolor sit amet</div>
+            <div class="description__content_text-bold">Donec nec justo eget</div>
+            <div class="text-gray">
```
Comment (mhryshkin) [inline index.html:36]: @OlegAdamovich Don't you find the name a bit [weird ](https://ru.bem.info/methodology/quick-start/#%D0%B1%D0%BB%D0%BE%D0%BA)for a block? Did you check If you need `text-gray` with this color and other styles elsewhere in the document?

File: index.html (hunk 5 above)
Comment (mhryshkin) [reply index.html]: @OlegAdamovich It's better to use classes everywhere possible, any thoughts on how to call a class for a body tag?

Comment (OlegAdamovich) [reply index.html]: Added class to `body` tag.

Comment (OlegAdamovich) [reply index.html]: Rename class to `page`.

File: index.html (hunk 3 above)
Comment (OlegAdamovich) [reply index.html]: Redone as element `description__content-text`.

File: index.html (hunk 10 above)
Comment (OlegAdamovich) [reply index.html]: Redone as element `description__content-text.`

**Comment by mhryshkin** [comment]
👍

**Pull request merged**